_redirects

```diff
@@ -1,13 +1,47 @@
 /cats /dogs
 
-/some-url  /some-other-url?x=yz
+/no-param-301 /index.html
 
-/no-params-to-function-301 /.netlify/functions/hello
+/param-in-to-301 /index.html?dog=silly
```

Comment:
/param-in-to-301a frog=:frog /index.html?dog=silly

/param-in-to-301b bog=:bog /index.html?dog=silly? bog=:bog